### DISCUSSION TITLE
How can I build the theme and push the output files to a different repo branch?

### DISCUSSION THREAD

**Discussion by albertpratomo** [uicrooks/shopify-theme-lab#141] [Q&A] [answered]
Shopify mentioned in this [doc](https://shopify.dev/themes/best-practices/version-control#separate-source-code-and-compiled-code-using-branches-recommended) that it's best to have a separate repo branch where all built files are stored. Then we can use  Shopify Github Integration to connect store with that branch.

Do you have any suggestion or maybe Github Action that can help doing this?

**Comment by sergejcodes** [comment, marked as answer]
It's easier having a separate repo dedicated to the dist files than a dist branch on the src repo. It's mentioned [here in the docs](https://themelab.uicrooks.com/guide/deployment.html#github-integration). Maybe a combination with a bash script for copying the dist files will do the trick.
